.huskyrc.js

```diff
@@ -0,0 +1,14 @@
+const tasks = arr => arr.join(' && ')
+
+module.exports = {
+  'hooks': {
+    'pre-commit': tasks([
+      'exec < /dev/tty',
+      `if ! eMSG=$(circleci config validate -c .circleci/config.yml); then
```

Comment:
Can we print a nicer error message if the circleci tool doesn't exist?

Reply:
bash printed out a message for it, is it not enough?
`Daniels-MacBook-Pro:analytics-node daniel.jackins$ git commit -m "test"
husky > pre-commit (node v8.16.0)
/bin/sh: circleci: command not found
CircleCI Configuration Failed Validation.

husky > pre-commit hook failed (add --no-verify to bypass)`

Reply:
Since we are making having the tool installed locally a requirement, I think ideally the message should print something like "Could not find `circleci` tool, please install it from https://circleci.com/docs/2.0/local-cli/".

As a new person contributing to this repo (especially external contributors), I would have no idea what this tool is. I would also okay with not validating this locally and forcing installing the CLI tool, as long as the CI is validating it. This would make it friendlier for new contributors, as they would not not have to go out of their way to install the CLI tool.

Reply:
It could be kind of nice to have the local validation as well to catch the errors before you've pushed the change and wait for circleci to catch it, though. Already saved me from pushing a change that had a tab instead of spaces :)

But if the hassle outweighs the benefit I can take out the local validation

Reply:
(and they could always use the --no-verify if they just want to skip the local validation)